examples/AeroSharp.Examples/Maps/MapCustomClassExample.cs

```diff
@@ -0,0 +1,66 @@
+﻿using AeroSharp.Connection;
+using AeroSharp.DataAccess;
+using AeroSharp.DataAccess.MapAccess;
+using AeroSharp.Examples.Utilities;
+using MessagePack;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace AeroSharp.Examples.Maps;
```

Comment:
I'd love to get the repo swapped to file-scoped namespaces as a whole, since I think it really improves readability. I also think that file-scoped namespaces will be the default in the future. 

What are your thoughts on leaving this as-is and having a follow up task to swap to file-scoped namespaces across the repo?

Reply:
Sounds good to me.